exporter.go

```diff
@@ -267,7 +267,7 @@ func scrapeMySQLGlobal(db *sql.DB, ch chan<- prometheus.Metric) error {
 	return rows.Err()
 }
 
-const mySQLconnectionPoolQuery = "SELECT hostgroup, srv_host, srv_port, * FROM stats_mysql_connection_pool"
```

Comment:
Code below assumes that those three columns are in that order: https://github.com/percona/proxysql_exporter/blob/90da3e31c43d9eed7575b4950162840723baadc9/exporter.go#L314-L320

There is a Docker image that can be used for testing: https://hub.docker.com/r/percona/proxysql2

Will you work on it? If not, who will?

Reply:
Here is a source repo for Dockerfile: https://github.com/percona/percona-docker/tree/master/proxysql